userspace/libscap/linux/scap_fds.c

```diff
@@ -721,7 +721,7 @@ int32_t scap_fd_read_ipv4_sockets_from_proc_fs(const char *dir, int l4proto, sca
 	f = fopen(dir, "r");
 	if(NULL == f)
 	{
-		ASSERT(false);
+		/*ASSERT(false);*/
```

Comment:
@erthalion So the goal here is to be able to test with a DEBUG build, is this right ?